src/fireblocks-sdk.ts

```diff
@@ -1868,61 +1912,90 @@ export class FireblocksSDK {
      * Call contract write function
      * @param blockchainId
      * @param templateId
+     *
+     * @returns WriteCallFunctionResponseDto
      */
     public async writeContractCallFunction(blockchainId: string, contractAddress: string, payload: WriteCallFunctionDto): Promise<WriteCallFunctionResponseDto> {
         return await this.apiClient.issuePostRequest(`/v1/contract-service/contract/${blockchainId}/${contractAddress}/function/write`, payload);
     }
 
     /**
-     * Issue a new token and link it to the tenant
-     * @param payload
+     * Creates a new asset and links it as a token.
+     *
+     * @param {IssueTokenRequest} payload - The payload containing information for token issuance.
+     *
+     * @returns {CreateTokenResponseDto} Response with created token ID
      */
-    public async issueNewToken(payload: IssueTokenRequest): Promise<PendingTokenLinkDto> {
+    public async issueNewToken(payload: IssueTokenRequest): Promise<CreateTokenResponseDto> {
```

Comment:
Wait... why not return `TokenLinkDto`? with Pending status? how will ripple and stellar work? you need to do another request?

Reply:
It's possible

Let's decide it tm and have the right change for it 🙏